tests/Application/src/Migrations/Version20191119131635.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20191119131635 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE sylius_channel ADD type VARCHAR(255) DEFAULT NULL');
```

Comment:
There's a migration added that adds `type` to `sylius_channel`, but I'm missing the one that reverts it (since we decided to remove that feature before 1.7 release).

Reply:
I've added only the migrations that were mentioned in UPGRADE file in Sylius/Sylius, I'll check that

Reply:
Dropping channel type is in the second migrations: [here](https://github.com/Sylius/InvoicingPlugin/pull/177/files#diff-6a3e796fc42923af8c0b713855b0d2dcR38)